libultraship/libultraship/InputEditor.cpp

```diff
@@ -93,7 +93,7 @@ namespace Ship {
 				if (ControllerName != "Keyboard" && ControllerName != "Auto") {
 					ControllerName += "##"+std::to_string(i);
 				}
-				if (ImGui::Selectable(ControllerName.c_str(), i == controlDeck->GetVirtualDevice(CurrentPort))) {
+				if (ImGui::Selectable(controlDeck->GetPhysicalDevice(i)->GetControllerName().c_str(), i == controlDeck->GetVirtualDevice(CurrentPort))) {
```

Comment:
This should also be done with the current physical index instead of the `ControllerName`.
```suggestion
				std::string DeviceName = controlDeck->GetPhysicalDevice(i)->GetControllerName();
				if (DeviceName != "Keyboard" && DeviceName != "Auto") {
					DeviceName += "##"+std::to_string(i);
				}
				if (ImGui::Selectable(DeviceName.c_str(), i == controlDeck->GetVirtualDevice(CurrentPort))) {
```

Reply:
implemented the suggested change, how would you suggest using the current physical index instead of controllername here? would you suggest reordering the devices in `ControlDeck::ScanPhysicalDevices` so Keyboard is always on top? in the current state, since we can have multiple controllers connected, the index of the keyboard is not something we can rely on.

Reply:
Oh sorry, I meant with the name of the current physical device. So basically `controlDeck->GetPhysicalDevice(i)->GetControllerName();` (what is done in the suggestion).

Reply:
ok cool!